src/allotropy/parsers/chemometec_nc_view/chemometec_nc_view_parser.py

```diff
@@ -0,0 +1,33 @@
+from allotropy.allotrope.models.adm.cell_counting.benchling._2023._11.cell_counting import (
+    Model,
+)
+from allotropy.allotrope.schema_mappers.adm.cell_counting.benchling._2023._11.cell_counting import (
+    Data,
+    Mapper,
+)
+from allotropy.named_file_contents import NamedFileContents
+from allotropy.parsers.chemometec_nc_view.chemometec_nc_view_reader import (
+    ChemometecNcViewReader,
+)
+from allotropy.parsers.chemometec_nc_view.chemometec_nc_view_structure import (
+    create_measurement_groups,
+    create_metadata,
+)
+from allotropy.parsers.chemometec_nc_view.constants import DISPLAY_NAME
+from allotropy.parsers.release_state import ReleaseState
+from allotropy.parsers.utils.pandas import map_rows
+from allotropy.parsers.vendor_parser import MapperVendorParser
+
+
+class ChemometecNcViewParser(MapperVendorParser[Data, Model]):
+    DISPLAY_NAME = DISPLAY_NAME
+    RELEASE_STATE = ReleaseState.WORKING_DRAFT
```

Comment:
I thought this was to be updated to RECOMMENDED?

Reply:
For some reason a unitest was failing when I had it in REC state, I just try again and it worked!